eval/alpaca_farm/run_eval.py

```diff
@@ -132,6 +132,26 @@ def main(args):
     # save to json
     with open(os.path.join(args.save_dir, f"metrics.json"), "w") as fout:
         json.dump(df_leaderboard.to_dict(), fout)
+
+    if args.upload_to_hf is not None:
+        # upload metrics to HF. Main metric is the LC winrate
+        # we divide by 100 to match other metrics.
+        results = df_leaderboard.to_dict()
+        if os.getenv("IS_ALPACA_EVAL_2", False) != "False":
```

Comment:
Lol, eek

Reply:
is there a nicer way to do this? this is mostly inherited from what AlapcaEval does internally (env flag to determine what version runs)...

Reply:
doesn't seem like it, I just hadn't seen alpacaeval internals.